migration-engine/migration-engine-tests/src/assertions.rs

```diff
@@ -52,6 +53,17 @@ impl SchemaAssertion {
         self.schema
     }
 
+    #[track_caller]
+    fn find_view_option<'a>(&'a self, view_name: &str) -> Option<ViewWalker<'a>> {
+        self.schema.view_walkers().find(|v| {
+            if self.tags.contains(Tags::LowerCasesTableNames) {
+                v.name().eq_ignore_ascii_case(view_name)
+            } else {
+                v.name() == view_name
+            }
+        })
```

Comment:
That made me think of something I (probably we) don't know: does the automatic table name lowercasing on MySQL apply to views as it does to tables?